ui/admin/tests/acceptance/aliases/update-test.js

```diff
@@ -71,26 +59,27 @@ module('Acceptance | aliases | update', function (hooks) {
 
   test('users can update an exisiting alias', async function (assert) {
     await visit(urls.aliases);
-    await click(`[href="${urls.alias}"]`);
-    await click(BUTTON_SELECTOR, 'Click edit mode');
-    await fillIn(NAME_FIELD_SELECTOR, NAME_FIELD_TEXT);
-    await click(SAVE_BTN_SELECTOR);
-    assert.strictEqual(instances.alias.name, NAME_FIELD_TEXT);
+    await click(commonSelectors.HREF(urls.alias));
+    await click(commonSelectors.EDIT_BTN);
+    await fillIn(commonSelectors.FIELD_NAME, commonSelectors.FIELD_NAME_VALUE);
+    await click(commonSelectors.SAVE_BTN);
+    
+    assert.strictEqual(instances.alias.name, commonSelectors.FIELD_NAME_VALUE);
   });
 
   test('can cancel changes to an existing alias', async function (assert) {
     const name = instances.alias.name;
     await visit(urls.aliases);
+    
+    await click(commonSelectors.HREF(urls.aliases));
+    await click(commonSelectors.HREF(urls.alias));
 
-    await click(`[href="${urls.aliases}"]`);
-    await click(`[href="${urls.alias}"]`);
-
-    await click(BUTTON_SELECTOR, 'Click edit mode');
-    await fillIn(NAME_FIELD_SELECTOR, NAME_FIELD_TEXT);
+    await click(commonSelectors.EDIT_BTN);
+    await fillIn(commonSelectors.FIELD_NAME, commonSelectors.FIELD_NAME_VALUE);
 
-    await click(BUTTON_SELECTOR, 'Click cancel');
+    await click(commonSelectors.CANCEL_BTN);
 
-    assert.dom(NAME_FIELD_SELECTOR).hasValue(`${name}`);
+    assert.dom(commonSelectors.FIELD_NAME).hasValue(`${name}`);
```

Comment:
nitpick: Could we remove the string interpolation? Looks like we can just use `name` as is.

Reply:
Fixed